spec/examples/schema_adapter_examples.rb

```diff
@@ -120,10 +126,12 @@
       subject.switch(schema1) do
         expect(connection.schema_search_path).to start_with %("#{schema1}")
         expect(User.sequence_name).to eq "#{schema1}.#{User.table_name}_id_seq"
```

Comment:
Based on the expectations and my understanding of your proposed solution, shouldn't we update this test to ensure that User's sequence name does not have the schema set inside nor outside the block?
@fsateler